SWLOR.Game.Server/Feature/StatusEffectDefinition/TutaminisStatusEffectDefinition.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.SkillService;
+using SWLOR.Game.Server.Service.AbilityService;
+using SWLOR.Game.Server.Service.StatusEffectService;
+using Player = SWLOR.Game.Server.Entity.Player;
+
+namespace SWLOR.Game.Server.Feature.StatusEffectDefinition
+{
+    public class TutaminisStatusEffectsEffectDefinition : IStatusEffectListDefinition
+    {
+        public Dictionary<StatusEffectType, StatusEffectDetail> BuildStatusEffects()
+        {
+            var builder = new StatusEffectBuilder();
+            Tutaminis1(builder);
+            Tutaminis2(builder);
+            Tutaminis3(builder);
+            Tutaminis4(builder);
+            Tutaminis5(builder);
+
+            return builder.Build();
+        }
+
+        private void Tutaminis1(StatusEffectBuilder builder)
+        {
+            builder.Create(StatusEffectType.Tutaminis1)
+                .Name("Tutaminis I")
+                .EffectIcon(EffectIconType.DamageResistance)
```

Comment:
I'm making an assumption that could be wrong here - but just so you're aware, this only affects which icon is displayed. This doesn't actually give damage resistance. Unsure if that's what you wanted or not, but thought I'd mention it.